topics/testing.md

```diff
@@ -0,0 +1,15 @@
+# Testing
```

Comment:
I'm a little confused why this is showing as a new file, did you update your master before creating the new branch?

Reply:
Actually, i did'nt.